akka-persistence-typed/src/test/java/akka/persistence/typed/javadsl/PersistentActorJavaDslTest.java

```diff
@@ -58,36 +60,36 @@
   interface Command extends Serializable {
   }
 
-  public static class Increment implements Command {
-    private Increment() {
-    }
-
-    public static Increment instance = new Increment();
+  public enum Increment implements Command {
+    INSTANCE
```

Comment:
This idiom of using `enum` for a singleton object is recommended in "Effective Java". One benefit is that it preserves the singleton-instance constraint when deserializing, which the existing implementation does not. It's also a little more concise (and a lot more concise than something that does explicitly handle deserialization).

Reply:
👍

Reply:
added that to the style guide also, thanks https://github.com/akka/akka/issues/22805

when the time is right we will do a swipe and adjust all our doc and test code to follow the idioms collected in the style guide (but of course it's best if we start using them right away)